dvc.yaml

```diff
@@ -18,6 +18,7 @@ stages:
     - input/land_site_rate_data.parquet
     - input/training_data.parquet
 
+    frozen: true
```

Comment:
This got dumped in https://github.com/ccao-data/model-res-avm/pull/240/files for some reason.